templates/navigation-bar.twig

```diff
@@ -0,0 +1,129 @@
+{% autoescape 'html' %}
```

Comment:
Do we need this tag? I thought the autoescape is setup in `functions.php`.

Reply:
That was there because we haven't yet added the timber autoescape in funcitons.php.
Was waiting for this to get merged to avoid coflicts.
To save time I suggest I enable Timber's autoescape and remove Twig's autoescape in this PR and together, if that is ok with you.. its practically 2 extra lines of changes. @dashaluna

Reply:
@angtheod I'd merge the Timber autoescape as a separate PR first, it's better to use as clean as possible git history to reverting purposes or what not. Hope that's ok?

Reply:
Ok, I did the changes for this PR. I will make a PR with the Timber autoescape after this one is merged.